inc/class-markdownparser.php

```diff
@@ -58,7 +62,29 @@ protected function inlineLink( $data ) {
 
 		// Override href.
 		$slug = get_slug_from_path( $root, $resolved );
-		$url = add_query_arg( [ 'id' => urlencode( $slug ) ] );
+		$url = get_url_for_page( UI\get_current_group_id(), $slug );
```

Comment:
Ideally, this should pull the group ID from the page instead, but we're not passing this around with the page currently.